checkout/other-features.md

```diff
@@ -0,0 +1,1401 @@
+
+{:.table .table-striped}
+| Field         | Type     | Description                                                    |
+| :------------ | :------- | :------------------------------------------------------------- |
+| `id`          | `string` | {% include field-description-id.md %}                          |
+| `redirectUrl` | `string` | The URI the user will be redirect to after a canceled payment. |
+
+### `onPaymentTransactionFailed`
+
+This event triggers when a payment attempt fails, further attempts can be made
+for the payment. An error message will appear in the payment UI, and the
+consumer will be able to try again or choose another payment instrument. The
+`onPaymentTransactionFailed` event is raised with the following event argument
+object:
+
+{:.code-header}
+**`onPaymentTransactionFailed` event object**
+
+```js
+{
+    "id": "/psp/paymentorders/payments/{{ page.payment_id }}",
+    "details": "[HttpCode ProblemTitle]"
+}
+```
+
+{:.table .table-striped}
+| Field     | Type     | Description                                         |
+| :-------- | :------- | :-------------------------------------------------- |
+| `id`      | `string` | {% include field-description-id.md %}               |
+| `details` | `string` | A human readable and descriptive text of the error. |
+
+### `onPaymentFailed`
+
+This event triggers when a payment has failed, disabling further attempts to
+perform a payment. The `onPaymentFailed` event is raised with the following
+event argument object:
+
+{:.code-header}
+**`onPaymentFailed` event object**
+
+```js
+{
+    "id": "/psp/paymentorders/payments/{{ page.payment_id }}",
+    "redirectUrl": "https://en.wikipedia.org/wiki/Failed"
+}
+```
+
+{:.table .table-striped}
+| Field         | Type     | Description                                                  |
+| :------------ | :------- | :----------------------------------------------------------- |
+| `id`          | `string` | {% include field-description-id.md %}                        |
+| `redirectUrl` | `string` | The URI the user will be redirect to after a failed payment. |
+
+### `onPaymentTermsOfService`
+
+This event triggers when the user clicks on the "Display terms and conditions"
+link. The `onPaymentTermsOfService` event is raised with the following event
+argument object:
+
+{:.code-header}
+**`onPaymentTermsOfService` event object**
+
+```js
+{
+    "origin": "owner | merchant",
+    "openUrl": "https://example.org/terms.html"
+}
+```
+
+{:.table .table-striped}
+| Field     | Type     | Description                                                                             |
+| :-------- | :------- | :-------------------------------------------------------------------------------------- |
+| `origin`  | `string` | `owner`, `merchant`. The value is always `merchant` unless Swedbank Pay hosts the view. |
+| `openUrl` | `string` | The URI containing Terms of Service and conditions.                                     |
+
+### `onError`
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (1022:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5659803)